src/uvwasi.c

```diff
@@ -1156,12 +1155,12 @@ uvwasi_errno_t uvwasi_fd_readdir(uvwasi_t* uvwasi,
       /* Write dirent to the buffer. */
       available = buf_len - *bufused;
       size_to_cp = sizeof(dirent) > available ? available : sizeof(dirent);
-      memcpy(buf + *bufused, &dirent, size_to_cp);
+      memcpy((char*)buf + *bufused, &dirent, size_to_cp);
```

Comment:
Not sure I got this right. But the original version will not build on visual studio compiler.